https://issues.redhat.com/browse/ACM-24225\ Dual-stack clust no longer needs Globalnet (#8389)
open-cluster-management/rhacm-docs: release_notes/acm_new.adoc

[#networking-new-features]
== Networking

> You can now deploy dual-stack clusters with OVN-Kubernetes Container Network Interface on Submariner without Globalnet.

Author: ```suggestion
- You can now deploy dual-stack clusters with OVN-Kubernetes Container Network Interface on Submariner without Globalnet.
```

> - You can now deploy Submariner without using Globalnet. For more details, see link:../networking/subm_configure.adoc#configuring-submariner[Configuring Submariner].

Author: ```suggestion
You can now deploy dual-stack clusters with OVN-Kubernetes Container Network Interface on Submariner without Globalnet.
```

> - You can now use dual-stack clusters with OVN-Kubernetes Container Network Interface on Submariner without Globalnet. For more details, see link:../networking/subm_configure.adoc#configuring-submariner[Configuring Submariner].

Author: ```suggestion
- You can now deploy Submariner without using Globalnet. For more details, see link:../networking/subm_configure.adoc#configuring-submariner[Configuring Submariner].
```

> 

Author: Do we need to change anything in the [Globalnet section](https://github.com/stolostron/rhacm-docs/blob/2.15_stage/networking/submariner/subm_configure.adoc#globalnet) ?

Reviewer: No changes to Globalnet as IPv6/dualstack is only applicable to non globalnet deployments.
